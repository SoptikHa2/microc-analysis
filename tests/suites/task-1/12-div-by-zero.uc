main() {
  var x, y;
  x = 42;
  y = 0;
  return x / y;
}
