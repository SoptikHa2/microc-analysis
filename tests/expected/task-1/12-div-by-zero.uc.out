---
test: run suites/task-1/12-div-by-zero.uc
---
microc: At "suites/task-1/12-div-by-zero.uc" (line 5, column 9): Division by zero
CallStack (from HasCallStack):
  error, called at src/Interpreter/Interpret.hs:21:18 in apr-semestral-0.1.0.0-7AYOm87uI8G3bgmoKhgVM9:Interpreter.Interpret

exit: 1
